data_processing/preprocess_csv.py

```diff
@@ -1,30 +1,13 @@
 import argparse
 import pandas as pd
+from omegaconf import OmegaConf
 
 from util import remove_noise, split_sentences
-from util import tokenize_and_normalize
+from util import tokenize_and_normalize, CONFIG_PATH
 
 
 def parse_arguments():
     parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "--full",
-        dest="full",
-        action="store",
-        help="The path to the full dataset to be preprocessed",
-    )
-    parser.add_argument(
-        "--train",
-        dest="train",
-        action="store",
-        help="The path to the file where the user wants " "to save the train portion of preprocessed " "dataset",
-    )
-    parser.add_argument(
-        "--test",
-        dest="test",
-        action="store",
-        help="The path to the file where the user wants " "to save the test portion of preprocessed " "dataset",
-    )
     parser.add_argument(
```

Comment:
This argument can also be stored in the yaml config to make reproduction easier. By the way, you can move the comments from here to a sample configuration file (yaml supports comments) in order to ease on-boarding for newcomers.

Reply:
Moved to `config.yml`